uiServices/homepage.go

```diff
@@ -38,14 +43,15 @@ func getIndexPage() *chi.Mux {
 		htmlBytes, err := ioutil.ReadFile(absPath)
 
 		if err != nil {
-			logrus.Error("Unable to read file: ",absPath, err)
+			logrus.Error("Unable to read file: ", absPath, err)
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
 
 		w.Header().Set(contentTypeHeader, htmlType)
 		w.Write([]byte(htmlBytes))
 	})
+	router.HandleFunc("/configs", configsHandler)
```

Comment:
needs to match what's documented in the README.

I personally prefer /configurations